FPSCamera/Cam/FollowCam.cs

```diff
@@ -63,9 +68,19 @@ public override bool Validate()
         public override Utils.Infos GetTargetInfos() => _target.GetInfos();
 
         public override Positioning GetPositioning()
-            => _target.GetPositioning().Apply(_LocalOffset)
-                                       .Apply(Config.G.FollowCamOffset.AsOffSet)
-                                       .Apply(_inputOffset);
+        {
+            var pos = _target.GetPositioning();
+            var look = GetSmoothLookPos();
+            var dir = pos.position.DisplacementTo(look);
+            if (dir.SqrDistance >= minLookDistance * minLookDistance) {
+                pos.angle = Angle.Lerp(pos.angle, dir.AsLookingAngle(), angleFactor);
```

Comment:
minLookDistance is to turn off look ahead when vehicle has stopped.
I compare square values because its faster than taking square root (optimization).